relay-general/Cargo.toml

```diff
@@ -14,7 +14,8 @@ chrono = { version = "0.4.11", features = ["serde"] }
 cookie = { version = "0.12.0", features = ["percent-encode"] }
 debugid = { version = "0.7.2", features = ["serde"] }
 dynfmt = { version = "0.1.4", features = ["python", "curly"] }
-enumset = "1.0.1"
+# TODO: Move back to enumset once https://github.com/Lymia/enumset/pull/18 is merged
+wasmer_enumset = { version = "1.0.1" }
```

Comment:
might as well define an alias so you minimize the diff in the code

Reply:
the auto-derive has the crate name hardcoded so it doesn't actually work.

Reply:
Enumset 1.0.3 has been released which should contain the fix.